addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <templates>
     <div t-name="spreadsheet_dashboard.DashboardAction" class="o_action o_spreadsheet_dashboard_action o_field_highlight">
-        <ControlPanel display="controlPanelDisplay">
+        <ControlPanel display="controlPanelDisplay" hideGlobalFilters="state.activeDashboard?.isPlaceholder">
             <t t-set-slot="layout-actions">
                 <t t-set="status" t-value="state.activeDashboard and state.activeDashboard.status"/>
                 <FilterValue
```

Comment:
`state.activeDashboard?.isPlaceholder` should be added to the `t-if` of this component